pycryptobot.py

```diff
@@ -57,7 +58,10 @@
             state.last_buy_size = float(df[df.action == 'buy']['size'])
             state.last_buy_filled = float(df[df.action == 'buy']['filled'])
             state.last_buy_price = float(df[df.action == 'buy']['price'])
-            state.last_buy_fee = round(state.last_buy_filled * state.last_buy_price * app.getTakerFee(), 2)
+            if app.getExchange() == 'binance':
+                state.last_buy_fee = (state.last_buy_filled / 100) * app.getTakerFee()
```

Comment:
Did not get, we are dividing last_by_price by 100?

Reply:
https://dappgrid.com/binance-fees-explained-fee-calculation/

```
How to calculate Binance spot fees?

Now let’s move on to Binance fee calculation. It’s quite simple. Let’s say you have 200 USDT in your wallet and you will buy BTC, BNB or any other coin.

When you make a trade with 200 USDT, 0.2 USDT will be charged as a trading fee. The calculation is simple: 200/100×0.10.

You should first divide your order size (total) by 100 and then multiply it by your fee rate which is 0.10% in this case.

So, if you buy Bitcoin with 200 USDT, you will basically get $199.8 worth of Bitcoin. To calculate these fees, you can also use our Binance fee calculator:
```

Reply:
See your point and we are making redundant calculation because of multiplication in the getTakerFee() function.